tests/pytorch-operator-base_test.go

```diff
@@ -107,9 +107,10 @@ spec:
     spec:
       containers:
       - command:
-        - /pytorch-operator.v1beta2
+        - /pytorch-operator.v1
```

Comment:
Can you also add the service yaml to the test? See https://github.com/kubeflow/manifests/pull/110/files#diff-5f390b526dd8deee13f3c4c19f68385dR317

Reply:
done